app/src/main/java/org/lineageos/selfie/MainActivity.kt

```diff
@@ -51,6 +52,10 @@ import androidx.camera.view.video.OnVideoSavedCallback
 import androidx.camera.view.video.OutputFileResults
 import androidx.core.app.ActivityCompat
 import androidx.core.content.ContextCompat
+import androidx.core.location.LocationListenerCompat
+import androidx.core.location.LocationManagerCompat
+import androidx.core.location.LocationRequestCompat
+import androidx.core.location.LocationRequestCompat.QUALITY_BALANCED_POWER_ACCURACY
```

Comment:
Use LocationRequestCompat.QUALITY_BALANCED_POWER_ACCURACY instead

Reply:
I'd like to fix fused location first lol